README.md

```diff
@@ -1,15 +1,226 @@
+### 1. Prepare inputs
+
+To run this pipeline you will need the following inputs: 
+
+* Paired Tumor-Normal paired BAM files
+* Corresponding BAM index files  
+* Input sample sheet 
+
+This pipeline processes paired BAM files and is capable of processing multiple samples in parallel. BAM files are expected to be coordinate sorted and indexed (see [Fastq-to-BAM](https://github.com/Sydney-Informatics-Hub/Fastq-to-BAM) for an example of a best practice workflow that can generate these files).  
+
+You will need to create a sample sheet with information about the samples you are processing, before running the pipeline. This file must be **comma-separated** and contain a header and one row per sample. Columns should correspond to sampleID, BAM-N file-path, BAM-T file-path: 
+
+```csv
+sampleID,bam-N,bam-T 
+SAMPLE1,/data/Bams/sample1-N.bam,/data/Bams/sample1-T.bam
+SAMPLE2,/data/Bams/sample2-N.bam,/data/Bams/sample2-T.bam|
+``````
+
+When you run the pipeline, you will use the mandatory `--input` parameter to specify the location and name of the input file: 
+
+```
+--input /path/to/samples.csv
+```
+
+### 2. Prepare the reference materials 
+
+To run this pipeline you will need the following reference files:
+
+* Indexed reference genome in FASTA format 
+
+You will need to download and index a copy of the reference genome you would like to use. Reference FASTA files must be accompanied by a .fai index file. If you are working with a species that has a public reference genome, you can download FASTA files from the [Ensembl](https://asia.ensembl.org/info/data/ftp/index.html), [UCSC](https://genome.ucsc.edu/goldenPath/help/ftp.html), or [NCBI](https://www.ncbi.nlm.nih.gov/genome/doc/ftpfaq/) ftp sites. You can use our [IndexReferenceFasta-nf pipeline](https://github.com/Sydney-Informatics-Hub/IndexReferenceFasta-nf) to generate indexes. 
+
+When you run the pipeline, you will use the mandatory `--ref` parameter to specify the location and name of the reference.fasta file: 
+
+```
+--ref /path/to/reference.fasta
+```
+
+
+### 3. Clone this repository 
+
+Download the code contained in this repository with: 
+
+```
+git clone https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf
+```
+
+This will create a directory with the following structure: 
+```
+Somatic-shortV-nf/
+├── LICENSE
+├── README.md
+├── config/
+├── images/
+├── scripts/
+├── main.nf
+├── modules/
+├── PBS_gadi_runPipeline.sh
+└── nextflow.config
+```
+The important features are: 
+
+* **main.nf** contains the main nextflow script that calls all the processes in the workflow.
+* **nextflow.config** contains default parameters to use in the pipeline.
+* **modules** contains individual process files for each step in the workflow. 
+* **config** contains infrastructure-specific config files (this is currently under development)
+
+### 4. Run the pipeline 
+
+The most basic run command for this pipeline is: 
+
+```
+nextflow run main.nf --input  sample.csv --ref reference.fasta
+
+```
```

Comment:
Again, is this copied directly from GSV? It is currently missing key elements.

Reply:
Changed to this pipeline default command
nextflow run main.nf --input samples.csv --ref reference.fasta  --intervalList_path path_to_intervals --number_of_intervals integer_value